cou/utils/juju_utils.py

```diff
@@ -139,6 +139,7 @@ class COUMachine:
 
     machine_id: str
     hostname: str
+    apps: tuple[str]
```

Comment:
maybe: `(("my-nova-compute", "nova-compute"), ("nova-compute", "nova-compute"))`

Reply:
As we talked before, it would be good to have also the charm name on it